ASP.NET Core/Web/AspNetCoreTemplate.Web/Areas/Identity/Pages/Account/Manage/EnableAuthenticator.cshtml

```diff
@@ -34,13 +34,13 @@
             </p>
             <div class="row">
                 <div class="col-md-6">
-                    <form method="post">
+                    <form id="send-code" method="post">
```

Comment:
I see that we now add these HTML element IDs. Is there any particular reason for this?